requirements-testing.txt

```diff
@@ -4,8 +4,8 @@ garnett==0.7.1
 GitPython==3.1.11
 gsd==2.3.0
 matplotlib>=3.0.0
-MDAnalysis==1.0.0
 numpy==1.19.3
+MDAnalysis==1.0.0
```

Comment:
Was this change related to Windows tests? Revert (not alphabetical).

Reply:
Yep, see [my last comment above](https://github.com/glotzerlab/freud/pull/685#issuecomment-721193525), reverting it was indeed my intention.